cmd/swctl/app/cmd_dependency.go

```diff
@@ -747,3 +755,42 @@ func isUserRoot() (bool, error) {
 	return true, nil
 
 }
+func postInstall(cli Cli) []error {
+	var errorGroup []error
+	sudoName, err := logname(cli)
+	fmt.Fprintln(cli.Out(), sudoName)
+	// handling case when linux has no login name (e.g. container)
+	if err == nil {
```

Comment:
Unnecessary paragraph dive and hard to read error branch (image multiple dives like this to read happy path commands with multiple indentation as prefix and then problematic pairing of error handling to right failure command). So lets handle errors first and then have the happy path in normal/basic indentation. This can be applied to any structuring to make it more readable.